gateway-messages/src/lib.rs

```diff
@@ -138,6 +138,25 @@ pub enum RotSlotId {
     B,
 }
 
+/// Image slot name for SwitchDefaultImage on component STAGE0
+#[derive(
+    Debug, Clone, Copy, PartialEq, Eq, SerializedSize, Serialize, Deserialize,
+)]
+pub enum Stage0SlotId {
+    Stage0,
+    Stage0Next,
+}
+
+#[derive(
+    Debug, Clone, Copy, PartialEq, Eq, SerializedSize, Serialize, Deserialize,
+)]
+pub enum ComponentSlot {
+    /// Hubris flash slot
+    Rot(RotSlotId),
+    /// Bootloader flash slot
+    Stage0(Stage0SlotId),
+}
```

Comment:
Are these two enums used anywhere?

Reply:
There is no higher level code updating stage0 right now. There is code in Hubris. I need to check on its use.

Reply:
Nope. Thanks for spotting this cruft.